student-store-ui/.eslintrc.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
```

Comment:
Nice! Were you able to get this working? 

The best way to make use of eslint is to connect it to your editor. Here's an example in VSCode: 

<img width="1544" alt="Screen Shot 2022-07-05 at 3 32 31 PM" src="https://user-images.githubusercontent.com/1775324/177428156-313a380c-ed31-4a6a-9cd1-66713e8bf9fb.png">

[Prettier](https://prettier.io/docs/en/editors.html) is also a great thing to add to your project so you don't have to worry about formatting.